_topics/en/adaptor-signatures.md

```diff
@@ -0,0 +1,242 @@
+    adaptor signature, then BIP340 is going to fail on `H(P || R || m)`
+    since Alice computed her hash with `H(P || R + T || m)`.
+
+  - If Bob sets `Y=R + T` so that it matches `H(P || R + T || m)`, BIP340
+    is going to fail on the initial `Y` since Bob is providing `R + T`
+    rather than the needed `R`.
+
+  Therefore Bob can't use the adaptor signature as a final signature.
+  However, he can create his own signature using it.  This is similar to the
+  signature Alice created but Bob doesn't commit to `t` here, since Bob
+  doesn't know that value.  All variables here except `T` are different
+  for Bob than they were for Alice:
+
+      s = r + H(P || R + T || m) * p
+
+  Unlike Alice, Bob doesn't need to tweak his signature.  Bob's `s` is
+  not a valid signature because it commits to `r` and `R + T`, which
+  won't pass BIP340 verification for the same reasons previously described.
+  To be valid, the signature needs to commit to `r + t` and `R + T`,
+  which Bob can't produce since he doesn't know `t`.
+
+  Bob gives Alice his adaptor signature:
+
+      s, R, T
+
+  Alice already knew `T`, but `(s, R, T)` is a standard adaptor
+  signature so we use its full form.  Alice can produce a final
+  signature from that adaptor signature using the hidden `t` value that
+  only she knows so far:
+
+      (s + t) * G ?= R + T + H(P || R + T || m) * P
+
+  Alice uses the final signature to broadcast Bob's transaction that
+  pays her.  When Bob sees `(s + t)` onchain, he can learn the value of `t`:
+
+      t = (s + t) - s
+
+  He can then use `t` to solve the adaptor signature Alice gave him
+  earlier:
+
+      (s' + t) * G ?= R + T + H(P || R + T || m) * P
+
+  Bob uses that final signature to broadcast the transaction Alice
+  originally gave him.
+  </details><br>
+  </div>
+
+  ### Relationship to multiparty signatures
+
+  Adaptor signatures usually can't secure a contract entirely by
+  themselves.  For example, in the above description of a coinswap,
+  Alice could double spend her payment to Bob after she learned Bob's
+  final signature, or Bob could've tried the same in reverse (with more
+  difficulty since we assumed Alice's transaction had one confirmation).
+  This is typically addressed by combining adaptor signatures with
+  multiparty signatures.  For example, Alice deposits her money into an
+  address that requires signatures from both her and Bob to spend.  Now
+  Alice can provide Bob with an adaptor signature for her half of the
+  multiparty transaction, which Bob can accept with perfect safety
+  knowing that Alice couldn't double spend the transaction without a
+  signature from him.  This may additionally require a timelocked refund
+  option in case one party refuses to sign.
+
+  In the [schnorr signature scheme][topic schnorr signatures], adaptor
+  signatures are usually proposed to be combined with signature
+  aggregation schemes such as [MuSig][topic musig] to allow the final
```

Comment:
s/Musig/musig/ per style guide, idem for the link in line 239 below (and the topic page needs updating WRT this)

Reply:
Huh?  Style guide says:

> Proper nouns
> [...]
> MuSig (the pubkey and signature aggregation scheme. Don't refer to Musig, muSig or musig)

(I remember this one because I voted for lowercase "musig" but lost.  :-)

Reply:
Quite right; I misremembered (note bene, seems Blockstream calls it MuSig).